base62.py

```diff
@@ -40,6 +67,25 @@ def decode(b):
 
     return v
 
+
+def decodebytes(s):
+    """Decodes a bytestring of base62 data into a bytes object.
+
+    :param s: A bytestring to be decoded in base62
+    :rtype: bytearray
+    """
+    _check_bytes_type(s)
+
+    decoded = decode(s.decode(DEFAULT_ENCODING))
+    buf = []
+    while decoded > 0:
+        buf.append(decoded & 0xff)
+        decoded = int(decoded / 256)
+    buf.reverse()
+
+    return bytearray(buf)
```

Comment:
I think this should return a `bytes` object rather than a `bytearray`. [In python 3](http://stackoverflow.com/a/5901849/4651668), `bytes` will be immutable like `str`. Also `bytes` support the `.decode()` which will be useful for getting text from decoded content...

```python
>>> base62_encoded = encodebytes('§'.encode())
>>> base62_encoded
'Cxj'
>>> bytes(decodebytes(base62_encoded.encode())).decode()
'§'
```